### PR TITLE
Configuration sample in reference doc has wrong yaml formatting

### PR DESCRIPTION
**Pull request by weixsun**
This pr is just a modification of the code style of the yaml code block.

### CODE REVIEW

File: spring-boot-project/spring-boot-docs/src/docs/asciidoc/features/external-config.adoc [hunk 1]
```diff
@@ -780,13 +780,13 @@ This style of configuration works particularly well with the `SpringApplication`
...
 	my:
-		service:
-			remote-address: 192.168.1.1
-			security:
-				username: admin
-				roles:
-				  - USER
-				  - ADMIN
+	  service:
+	    remote-address: "192.168.1.1"
```
Comment (snicoll) [inline spring-boot-project/spring-boot-docs/src/docs/asciidoc/features/external-config.adoc:784]: This PR is doing two different things here. Are the quote around the value necessary?

File: spring-boot-project/spring-boot-docs/src/docs/asciidoc/features/external-config.adoc [hunk 2]
```diff
@@ -780,13 +780,13 @@ This style of configuration works particularly well with the `SpringApplication`
...
+	    security:
+	      username: "admin"
```
Comment (snicoll) [inline spring-boot-project/spring-boot-docs/src/docs/asciidoc/features/external-config.adoc:786]: Ditto

**Review by snicoll — COMMENTED** [review]
Thanks for the PR, I've left a couple comments.

File: spring-boot-project/spring-boot-docs/src/docs/asciidoc/features/external-config.adoc (hunk 1 above)
Comment (weixsun) [reply spring-boot-project/spring-boot-docs/src/docs/asciidoc/features/external-config.adoc:784]: Ha, I think the String type here is more suitable, so I just did it.

**Comment by snicoll** [comment]
Thanks again @weixsun. I have reverted the quote change as discussed.

**Comment by philwebb** [comment]
I think we should probably quote Strings in YAML. I tried to add a review note but I didn't click "submit" 🤦. I've opened #28709.

**Pull request closed without merging**